### ISSUE TITLE
feature Doesn't empty `to` in `g-link`

### ISSUE THREAD

**Issue by tyankatsu0105** [gridsome/eslint-plugin-gridsome#3]
**Please describe what the rule should do:**

`g-link` is required 'to'.
When 'to' is not exist, display error message.

**What category should the rule belong to?**

- [ ] Enforces code style
- [x] Warns about a potential error
- [ ] Suggests an alternate way of doing something
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**
```vue
<template>
    <g-link
    class="foo"
    exact
  >
</template>
```

This would be correct:
```vue
<template>
  <g-link
    to="/"
    class="foo"
    exact
  >
</template>
```

```vue
<template>
  <g-link
    :to="to.path"
    class="foo"
    exact
  >
</template>
```

**Additional context**
